app/javascript/controllers/maps_controller.js

```diff
@@ -0,0 +1,103 @@
+import { Controller} from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["field", "map", "latitude", "longitude"]
+
+  connect() {
+    if (typeof (google) != "undefined"){
+      this.initializeMap()
+    }
+  }
+
+  initializeMap() {
+    this.map()
+    this.marker()
+    this.autocomplete()
+    console.log('init')
```

Comment:
Remember, you should not leave test or debug values ​​on GitHub (as is the case with` console.log()`).

Your code should always be as clean as possible.

Reply:
Fixed!